templates/frontend/components/monographList.tpl

```diff
@@ -32,12 +32,20 @@
 			{assign var="isFeatured" value=false}
 		{/if}
 		{if $isFeatured}
-			{include file="frontend/objects/monograph_summary.tpl" monograph=$monograph isFeatured=$isFeatured}
+			{if $heading && $titleKey}
+				{include file="frontend/objects/monograph_summary.tpl" monograph=$monograph isFeatured=$isFeatured heading="h3"}
+			{else}
+				{include file="frontend/objects/monograph_summary.tpl" monograph=$monograph isFeatured=$isFeatured heading="h2"}
+			{/if}
```

Comment:
`{$heading}` can actually be _any_ level, not just `<h2>`. For example, I can call `monographList.tpl` like this:

```
{include file="frontend/components/monographList.tpl" heading="h4"}
```

So you need to account for that when defining the headings at the top of the template. Like this:

```
{if !$heading}
	{assign var="heading" value="h2"}
{/if}
{if !$titleKey}
	{assign var="monographHeading" value=$heading}
{elseif $heading == 'h2'}
	{assign var="monographHeading" value="h3"}
{elseif $heading == 'h3'}
	{assign var="monographHeading" value="h4"}
...and so on up to h5...
{/if}
```

Then you can pass `$monographHeading` down to the `monograph_summary.tpl` template. Once you've done that, go through and manually test it. Find a template that uses `monographList.tpl` and pass in different `heading` and `titleKey` values, and then check the heading levels of everything to see if it comes out right.

Reply:
Made this change in https://github.com/pkp/omp/pull/795/commits/56a8d296bf9d9a1a3a60a4ca933ac25c43163660